allocations/migrations/0007_charge.py

```diff
@@ -70,6 +70,10 @@ def _str_to_localtime(utctimestr):
 
 
 def init_charge_from_openstack_db(apps, schema_editor):
+    # No initial charges in test DB
```

Comment:
Is this just to skip migrations when running in CI?
Do we know why this needs to access an external resource to run a migration anyway?

Reply:
`settings.TESTING` is added in this PR, and is true when the code is run with `./manage.py test`.

This code initially loaded existing reservations at openstack sites into the charges database table.